packages/whale-api-client/src/api/poolpairs.ts

```diff
@@ -243,6 +243,11 @@ export interface AllSwappableTokensResult {
   swappableTokens: TokenIdentifier[]
 }
 
+export interface EstimatedDexFees {
+  tokenA: string
+  tokenB: string
```

Comment:
```suggestion
  tokenA: string // BigNumber
  tokenB: string // BigNumber
```

Thought these were referring to the token's name / symbol at first 😛